pages/learn/develop/hardware/usb.md

```diff
@@ -0,0 +1,53 @@
+---
+title: USB Interface
+---
+
+# USB
+
+* [balenaFin 1.x](/hardware/usb#balenaFin)
+* [Raspberry Pi Family](/hardware/usb#raspberry-pi-family)
+
+## BalenaFin
+On certain USB hubs, you can control the per port power - this is usually referred to PPPS i.e. per port power switching. We use the LAN9514 USB 2.0 hub + ethernet controller on the balenaFin 1.x - which supports this. The recommended way to do this is to use the [`uhubctl`](https://github.com/mvp/uhubctl) tool.
```

Comment:
```suggestion
On certain USB hubs, one can control the power per port. This is usually referred to as per port power switching or PPPS. BalenaFin 1.x uses the LAN9514 USB 2.0 hub + ethernet controller which supports PPPS. The recommended way to implement this is to make use of the [`uhubctl`](https://github.com/mvp/uhubctl) tool.
```

IMHO, docs are usually better structured as 3rd person with 2nd person used sparingly unless we are writing a tutorial or getting started. Where we are speaking them personally in a guiding step-by-step tone. Let me know if it makes sense. Otherwise, we can stick with 2nd person.